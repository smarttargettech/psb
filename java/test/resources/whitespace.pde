void setup() {
    size(640, 360); 
    stroke(255);
    noFill();
}

void drawBezier(int i) {
    bezier(
        mouseX - (i/2.0),
        40+i,
        410,
        20,
        440,
        300,
        240 - (i/16.0),
        300 + (i/8.0)
    );
}

void draw() {
    background(0);
    for (int i = 0; i < 200; i += 20) {
        drawBezier(i);
    }
}