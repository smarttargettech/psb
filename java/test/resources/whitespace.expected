import processing.core.*;
import processing.data.*;
import processing.event.*;
import processing.opengl.*;

import java.util.HashMap;
import java.util.ArrayList;
import java.io.File;
import java.io.BufferedReader;
import java.io.PrintWriter;
import java.io.InputStream;
import java.io.OutputStream;
import java.io.IOException;

public class whitespace extends PApplet {

public void setup() {
    /* size commented out by preprocessor */; 
    stroke(255);
    noFill();
}

public void drawBezier(int i) {
    bezier(
        mouseX - (i/2.0f),
        40+i,
        410,
        20,
        440,
        300,
        240 - (i/16.0f),
        300 + (i/8.0f)
    );
}

public void draw() {
    background(0);
    for (int i = 0; i < 200; i += 20) {
        drawBezier(i);
    }
}


    public void settings() { size(640, 360); }

    static public void main(String[] passedArgs) {
        String[] appletArgs = new String[] { "whitespace" };
        if (passedArgs != null) {
            PApplet.main(concat(appletArgs, passedArgs));
        } else {
            PApplet.main(appletArgs);
        }
    }
}
